environments/aws/rds.tf

```diff
@@ -14,8 +14,8 @@ module "flyte_db" {
   subnets             = module.vpc.database_subnets
   vpc_id              = module.vpc.vpc_id
 
-  database_name          = "flyteadmin"
-  master_username        = "flyteadmin"
+  database_name          = "app"
```

Comment:
The problem is that this setting is platform-specific. You can totally change it and, in turn, use a different db name and user name in your values file. But the base chart has to remain somewhat generic